files/nrpe/check_keystone

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+
+# TODO This needs to be updated
```

Comment:
Is the general idea that this TODO will be fixed "later"?

Reply:
Improved comment.